lib/mais_orcid_client.rb

```diff
@@ -57,19 +57,51 @@ def fetch_orcid_users(limit: nil, page_size: nil)
     end
   end
 
+  # Fetch a user details,  including scope and token, given either a SUNetID or ORCIDID
+  # @param [string] sunetid of user to fetch
+  # @param [orcid] orcidid of user to fetch (ignored if sunetid is also provided)
+  # @return [<OrcidUser>, nil] orcid user or nil if not found
+  def fetch_orcid_user(sunetid: nil, orcidid: nil)
+    raise "must provide either a sunetid or orcidid" unless sunetid || orcidid
+
+    sunetid ? fetch_by_sunetid(sunetid) : fetch_by_orcidid(orcidid)
+  end
+
```

Comment:
this existing method is extended to support fetching by sunetid or orcidid

the method signature remains identical for all consumers who search by sunetid, so changes needed for them